packages/daemon/src/daemon.js

```diff
@@ -651,6 +699,24 @@ const makeEndoBootstrap = (
     makeMailbox,
   });
 
+  const allowedInspectorFormulaTypes = [
+    'eval-id512',
+    'lookup-id512',
+    'make-unconfined-id512',
+    'make-bundle-id512',
+    'guest-id512',
+    'web-bundle',
+  ];
+
+  const allowedInspectorFormulaIdentificationsByType = {
+    eval: ['endowments', 'worker'],
```

Comment:
I’m not sure that inspectors generalize this well. `endowments` is an array of identifiers and we want the corresponding values, whereas `worker` is a single identifier. This does not mention `source` which is just a string.

In short, I think we should have a boring identify function for each formula type rather than a generalization because it will be more legible and easier to debug.